ui/lib/sync/addon/components/secrets/page/destinations/create-and-edit.hbs

```diff
@@ -12,27 +12,33 @@
 
   {{#each @destination.formFieldGroups as |fieldGroup|}}
     {{#each-in fieldGroup as |group fields|}}
-      {{#if (and (eq group "Credentials") (not @destination.isNew))}}
+      {{#if (not-eq group "default")}}
         <hr class="has-top-margin-xl has-bottom-margin-l has-background-gray-200" />
-        <Hds::Text::Display @tag="h2" @size="400" @weight="bold">Credentials</Hds::Text::Display>
-        <Hds::Text::Body @tag="p" @size="100" @color="faint" class="has-bottom-margin-m">
-          Connection credentials are sensitive information and the value cannot be read. Enable the input to update.
+        <Hds::Text::Display @tag="h2" @size="400" @weight="bold">{{group}}</Hds::Text::Display>
+        <Hds::Text::Body
+          @tag="p"
+          @size="100"
+          @color="faint"
+          class="has-bottom-margin-m"
+          data-test-destination-subText={{group}}
+        >
+          {{(this.groupSubtext group @destination.isNew)}}
```

Comment:
instead of passing in `isNew` here can we just access `this.args.destination.isNew` in the component helper?

Reply:
I tried that and then realized I was hitting issues because this is within an each iteration, so it's easier to access the destination by passing it in like this.